moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_planner.h

```diff
@@ -50,6 +50,7 @@ class ChompPlanner
   ChompPlanner();
   virtual ~ChompPlanner(){};
 
+  // TODO: switch API from moveit_msgs to MotionPlanRequest / MotionPlanDetailedResponse
```

Comment:
the TODO in this comment seems to have happened already

Reply:
True, e017669ef0c3e18cc02c49b2d60080a4cf581552 is an empty patch that was previously applied.

I will fix this later to give people a stable branch for the moment.